src/js/Controllers/RpcFactory.js

```diff
@@ -330,6 +330,7 @@ class RpcFactory {
                 "ccpu_version": "0.0.1",
                 "language": "EN-US",
                 "wersCountryCode": "WAEGB",
+                "systemHardwareVersion": "123.456.789",
```

Comment:
@ymalovanyi I would remove the `,` at the end of line 333 to make this a valid JSON.

Other methods in this class like `static UIGetCapabilitiesResponse(rpc)` on line 133 are doing something similar.  For this one, the last field in the JSON doesn't have a `,` at the end of the line (line 147).

Reply:
@santhanamk Removed trailing comma as you suggest